src/scheduler/node_info.c

```diff
@@ -1847,15 +1847,15 @@ collect_resvs_on_nodes(node_info **ninfo_arr, resource_resv **resresv_arr, int s
  * @param[in]	ninfo	-	the nodes to collect for
  * @param[in]	resresv_arr	-	the array of jobs to consider
  * @param[in]	size	-	the size (in number of pointers) of the job arrays
- * @param[in]   flag	-	flag to indicate whether to do ghost job detection
+ * @param[in]	flags	-	to indicate whether to do ghost job detection
  *
  * @retval	1	: upon success
  * @retval	2	: if a job reported on nodes was not found in the job arrays
  * @retval	0	: upon failure
  *
  */
 int
-collect_jobs_on_nodes(node_info **ninfo_arr, resource_resv **resresv_arr, int size, int flag)
+collect_jobs_on_nodes(node_info **ninfo_arr, resource_resv **resresv_arr, int size, enum misc_constants flags)
```

Comment:
Don't make this an enum, just make it an int or unsigned int.  If we ever add a new flag, it won't be in misc_constants.